crates/bevy_transform/src/components/transform.rs

```diff
@@ -233,31 +235,53 @@ impl Transform {
         self.rotate(Quat::from_rotation_z(angle));
     }
 
-    /// Rotates this [`Transform`] around its `X` axis by `angle` (in radians).
+    /// Rotates this [`Transform`] by the given `rotation`.
+    ///
+    /// The `rotation` is relative to this [`Transform`]'s current rotation.
+    #[inline]
+    pub fn rotate_local(&mut self, rotation: Quat) {
+        self.rotation *= rotation;
```

Comment:
Does this not have the same issue of order of multiplication as what was fixed?

Reply:
This order of multiplication is not inherently wrong.
It's incorrect for `rotate_around` as that rotates around a point relative to the origin of the transform(ie. the parent if present, otherwise the origin of the world) not a local point relative to the transform itself.